ext/gd/gd.c

```diff
@@ -1029,7 +1029,11 @@ zend_module_entry gd_module_entry = {
 	"gd",
 	gd_functions,
 	PHP_MINIT(gd),
+#if HAVE_GD_BUNDLED && HAVE_LIBFREETYPE
```

Comment:
Please make the function unconditional and only the font cache call conditional.

Reply:
Done. Note that RSHUTDOWN is conditionally declared/defined, though.